modules/ROOT/pages/troubleshooting-studio.adoc

```diff
@@ -3,4 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-== See Also
+Troubleshooting and errors you can receive when using Studio:
```

Comment:
I'm not sure what this means.

For more information about troubleshooting errors with Studio?

Reply:
I used the same sentence I found in [ACB docs ](https://docs.mulesoft.com/anypoint-code-builder/troubleshooting)(don't let Duke know about this hahahahaha)

Reply:
Oh